ee/secureenclaverunner/secureenclaverunner_test.go

```diff
@@ -0,0 +1,93 @@
+//go:build darwin
+// +build darwin
+
+package secureenclaverunner
```

Comment:
Would you add in a `Test_InterruptMultiple` test also? (You can basically just paste in https://github.com/kolide/launcher/blob/main/ee/control/consumers/remoterestartconsumer/remoterestartconsumer_test.go#L120.) I like to have a `Test_InterruptMultiple` everywhere we add a new rungroup to be certain we keep avoiding any regressions in the interrupt behavior.